testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/BuildMultiprocessTestPlayer.cs

```diff
@@ -0,0 +1,117 @@
+using System;
+using System.IO;
+#if UNITY_EDITOR
+using UnityEditor;
+using UnityEditor.Build.Reporting;
+#endif
+using UnityEngine;
+
+/// <summary>
+/// This is needed as Unity throws "An abnormal situation has occurred: the PlayerLoop internal function has been called recursively. Please contact Customer Support with a sample project so that we can reproduce the problem and troubleshoot it."
+/// when trying to build from Setup() steps in tests.
+/// </summary>
+public class BuildMultiprocessTestPlayer : MonoBehaviour
+{
+    public const string multiprocessBaseMenuName = "MLAPI Multiprocess Test";
+    public const string BuildAndExecuteMenuName = multiprocessBaseMenuName + "/Build - Execute multiprocess tests #%t";
+    public static string buildPath => Path.Combine(Path.GetDirectoryName(Application.dataPath), "Builds/MultiprocessTestBuild");
+    public const string mainSceneName = "MultiprocessTestingScene";
```

Comment:
What's the naming convention for const strings? Seems inconsistent here.

Reply:
this is super weird indeed. how come dotnet-format (standards check) didn't catch these?

Reply:
hmm, I guess standards check is not considering `testproject` but only looks at `MLAPI.XXX` stuff.

Reply:
Yeah, noticed that as well.

Reply:
looking at yamato logs, looks like it's testing testproject, but skipping certain csproj... will need to investigate

Reply:
fixed